addons/custom-block-text/addon.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Customizable block text style",
-  "description": "Allows you to adjust the size and thickness of block text or add a shadow to make it easier to read.",
+  "description": "Allows you to adjust the size and font weight of block text or add a shadow to make it easier to read.",
```

Comment:
"Size and font weight" sounds slightly inconsistent. I wonder if saying "font size and weight" or just "size and weight" would be better. I don't know, what do some other folks think?

Wait for some conversation before changing anything.